tests/LANraragi/Plugin/Metadata/Eze.t

```diff
@@ -18,11 +18,11 @@ require "$cwd/tests/mocks.pl";
 
 use_ok('LANraragi::Plugin::Metadata::Eze');
 
-note("eze Tests");
+note("eze-lite Tests, with save original title but no original title exist");
 {
     # Copy the eze sample json to a temporary directory as it's deleted once parsed
     my ( $fh, $filename ) = tempfile();
-    cp( $SAMPLES . "/eze/eze_sample.json", $fh );
+    cp( $SAMPLES . "/eze/eze_lite_sample.json", $fh );
```

Comment:
It'd be nice to refactor this entire part (mocks from a given filename + trap with save_title/origin_title/additional_tags set) into a separate function so it's not copy-pasted 4 times in the test code. 😀

Reply:
Sorry that I don't know the how to return multiple values in a function by Perl well.
The following commit tried some work, but failed.

https://github.com/Difegue/LANraragi/pull/606/commits/7bb2df58e42f14da15c2d55778691e45c83a0960

Test cases are correct, but it seems the test function is always got wrong parameters. I don't know where goes wrong.